incubator/virtualcluster/pkg/syncer/conversion/mutate.go

```diff
@@ -256,12 +259,40 @@ func omitDuplicates(strs []string) []string {
 	return ret
 }
 
+// TODO(zhuangqh): make this plugable.
+func (p *podMutator) attachTenantMeta(vPod *v1.Pod) error {
+	if len(vPod.ObjectMeta.OwnerReferences) == 0 {
```

Comment:
You cannot assume all Pods in tenant are deployment Pods. The Pods can be standard alone Pods and owned by any controller.

Reply:
done